### PR TITLE
chore: restructure quo2 components

### PR DESCRIPTION
**Pull request by J-Son89**
Fixes:
https://github.com/status-im/status-mobile/issues/13903

Restructured quo2 component library so that is has a 1-1 mapping with the Figma designs. 
https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Mobile-Components?node-id=1338%3A34482

Some files needed changing. 

Although this p.r is quite large no code was changed in it 👍

### CODE REVIEW

File: src/quo2/components/filter_tag.cljs [hunk 1]
```diff
@@ -3,8 +3,8 @@
             [quo.theme :as theme]
             [quo.react-native :as rn]
             [quo2.components.icon :as icons]
-            [quo2.components.text :as text]
-            [quo2.components.base-tag :as base-tag]))
+            [quo2.components.markdown.text :as text]
+            [quo2.components.tags.base-tag :as base-tag]))
 
 (def themes {:light {:default  {:border-color             colors/neutral-20
```
Comment (jo-mut) [inline src/quo2/components/filter_tag.cljs:9]: is there a reason why you didnt place this file in the tags directory?

Comment (jo-mut) [reply src/quo2/components/filter_tag.cljs:9]: you could also confirm that you didnt leave it out under the screens directory. It seems filter-tags are missing there

**Review by jo-mut — CHANGES_REQUESTED** [review]

File: src/quo2/components/filter_tag.cljs (hunk 1 above)
Comment (J-Son89) [reply src/quo2/components/filter_tag.cljs:9]: yeah I wanted to check about this component. I could not find it in the figma file. 
https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Mobile-Components?node-id=180%3A9668

Comment (jo-mut) [reply src/quo2/components/filter_tag.cljs:9]: this is it https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Mobile-Components?node-id=180%3A9685

Comment (J-Son89) [reply src/quo2/components/filter_tag.cljs:9]: ah I got it now, will adjust. Thanks!

**Review by flexsurfer — APPROVED** [review]

**Review by ibrkhalil — APPROVED** [review]

**Review by jo-mut — APPROVED** [review]

**Pull request merged**